Assets/SEE/UI/DebugAdapterProtocol/DebugAdapterProtocolSession.cs

```diff
@@ -301,24 +291,26 @@ protected void Start()
             }
             catch (Exception e)
             {
-                ConsoleWindow.AddMessage(e.Message + "\n", "Adapter", "Error");
-                Debug.LogWarning(e);
-                Destroyer.Destroy(this);
+                OnInitializationFailed(e.Message + "");
```

Comment:
Addition of the empty string doesn't have any effect (I think):
```suggestion
                OnInitializationFailed(e.Message);
```